django_ninja_crudl/types.py

```
@@ -1,32 +1,45 @@
 """Shared types for the CRUDL classes."""
 
+from collections.abc import Mapping, Sequence
```

Ruff (0.8.2)

`collections.abc.Mapping` imported but unused

Remove unused import

(F401)